README.md

```diff
@@ -27,10 +27,9 @@ The meeting settings for each channel can be configured in the Channel Header Dr
 
 Meeting settings include:
 
-- Schedule Day: Day of the week when the meeting is scheduled.
-- Hashtag Format: The format of the hashtag for the meeting date. The date format is based on [Go date and time formatting](https://yourbasic.org/golang/format-parse-string-time-date-example/#standard-time-and-date-formats).
-  The date format must be wrapped in double Braces ( {{ }} ).
-  A default is generated from the first 15 characters of the channel's name with the short name of the month and day (i.e. Dev-{{ Jan02 }}).
+- Schedule Day: Day of the week when the meeting is being scheduled.
+- Hashtag Format: The format of the hashtag for the meeting date. 
+  A default hashtag is generated from the first 15 characters of the channel's name with the short name of the month and day (i.e. Dev_{{ Jan 2 }}).
```

Comment:
Seems unrelated to this PR

Reply:
Please follow sequence for reviewing these three PRs  #66->#12->#47. There is one commit for each issue.